config/archaeology/su_humsci_theme.settings.yml

```diff
@@ -1,18 +0,0 @@
-features:
```

Comment:
This doesn't seem related, are you sure you want to remove this?

Reply:
Archaeology is now on the `archaeology` theme so these aren't even used now.